bedrock/settings/static_media.py

```diff
@@ -394,6 +394,26 @@
         ),
         'output_filename': 'css/firefox_new-bundle.css',
     },
+    'firefox_new_scene1': {
+        'source_filenames': (
+            'css/sandstone/sandstone-resp.less',
```

Comment:
Hm, I think I agree that 1 additional HTTP request on scene 1 (or when going directly to scene 2) is worth the cache savings on a scene 1 > scene 2 user flow.

Will create a `firefox_new_common` bundle.